<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>401</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>421</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>91</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/icon/icons8-account-100.png);
background-repeat:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/resources/img/icons8-account-100.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>187</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_edit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>127</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(106, 138, 130);
border-radius: 14px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_edit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>253</y>
       <width>127</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(106, 138, 130);
border-radius: 14px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="teacher_student_login">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>120</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="response_label">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>300</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(167, 65, 74);</string>
     </property>
     <property name="text">
      <string>Username or Password is Incorrect, please try again!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="login_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>121</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Display</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-radius:10px;
}
QPushButton:hover{
	background-color: rgb(106, 138, 130);
	color:rgb(255, 255, 255)
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/resources/img/icons8-login-48.png</normaloff>:/resources/img/icons8-login-48.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
